packages/matrix/tests/objectives.spec.ts

```diff
@@ -64,7 +64,7 @@ test.describe('Room objectives', () => {
       'Objective: Make sure that all room members greet each other by saying "Hello"',
     );
     await assertMessages(page, [
-      { from: 'user1', message: 'user1 has set the room objectives' },
```

Comment:
This is down to the change of message rendering, objectives aren't in the AI assistant design.